sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -19,42 +20,30 @@
 
 from ._deserialize import _convert_to_entity, _trim_service_metadata
 from ._entity import TableEntity
-from ._error import (
-    _decode_error,
-    _process_table_error,
-    _reprocess_error,
-    _reraise_error,
-    _validate_tablename_error
-)
-from ._generated.models import (
-    SignedIdentifier,
-    TableProperties
-)
+from ._error import _decode_error, _process_table_error, _reprocess_error, _reraise_error, _validate_tablename_error
+from ._generated.models import SignedIdentifier, TableProperties
 from ._serialize import _get_match_headers, _add_entity_properties, _prepare_key
 from ._base_client import parse_connection_str, TablesBaseClient
 from ._serialize import serialize_iso, _parameter_filter_substitution
 from ._deserialize import deserialize_iso, _return_headers_and_deserialized
 from ._table_batch import TableBatchOperations, EntityType, TransactionOperationType
-from ._models import (
-    TableEntityPropertiesPaged,
-    UpdateMode,
-    TableAccessPolicy,
-    TableItem
-)
+from ._models import TableEntityPropertiesPaged, UpdateMode, TableAccessPolicy, TableItem
 
 if TYPE_CHECKING:
     from azure.core.credentials import AzureNamedKeyCredential, AzureSasCredential
 
 
-class TableClient(TablesBaseClient): # pylint: disable=client-accepts-api-version-keyword
+class TableClient(TablesBaseClient):  # pylint: disable=client-accepts-api-version-keyword
     """A client to interact with a specific Table in an Azure Tables account.
 
     :ivar str account_name: The name of the Tables account.
     :ivar str table_name: The name of the table.
     :ivar str url: The full URL to the Tables account.
     """
 
-    def __init__(self, endpoint: str, table_name: str, **kwargs: Any) -> None: # pylint: disable=missing-client-constructor-parameter-credential
+    def __init__(
+        self, endpoint: str, table_name: str, **kwargs: Any
+    ) -> None:  # pylint: disable=missing-client-constructor-parameter-credential
```

Comment:
Can we please update the constructor to fix this pylint diable statement?
`def __init__(self, endpoint: str, table_name: str, * credential: Union[...] = None, **kwargs: Any) -> None`

Reply:
I'll update but without default value `None`, because Pylance complains `parameter with "*" or "**" cannot have default value`.

Reply:
I think it should have a comma after `*`, like in async: https://github.com/Azure/azure-sdk-for-python/blob/5eef0a2f55d7fa12894b7ace041d11087d72a268/sdk/tables/azure-data-tables/azure/data/tables/aio/_table_client_async.py#L52-L59

Reply:
Yup - sorry that was my typo :D

Reply:
Pylint still failing with the same error `missing-client-constructor-parameter-credential` with the fix. Should we move `credential` parameters instead of keywords? @annatisch

Reply:
I have moved `credential` to params, please take a look. @annatisch

Reply:
Yeah I don't think that pylint rule is smart enough to check for `credential` as a keyword-only parameter so you'll need to continue to ignore it for now. @annatisch - do we want to update the rule to let this pass?